api-reference/10 UI Components/dxChart/1 Configuration/argumentAxis/breaks/breaks.md

```diff
@@ -14,7 +14,7 @@ A scale break allows breaking off a part of the axis to improve the readability
 
```

Comment:
Is it possible to update line 13 above? I'd like to avoid the -ing form ("breaking"), and update this scale break explanation. My suggestion is:

`A scale break is an area across an axis that is displayed instead of a section of an axis range. It allows you to improve the readability of a chart section with a large gap in its range.`

Let me know what you think, and feel free to suggest an alternative as well.

Reply:
Hello, your example seems valid. I would only fix singular: 'readability of chart sections with large gaps in their range'.

Reply:
Thank you. Sounds good. Can you please apply the changes?